action.yml

```diff
@@ -18,9 +18,12 @@ inputs:
   api-private-key:
     description: 'The PKCS8 format Private Key for AppStore Connect API'
     required: true
+  release-note:
```

Comment:
Can you change this to `release-notes` to match the code and common way people describe this?